src/coreclr/src/jit/emit.cpp

```diff
@@ -5193,7 +5281,57 @@ unsigned emitter::emitEndCodeGen(Compiler* comp,
 
         for (unsigned cnt = ig->igInsCnt; cnt; cnt--)
         {
+#ifdef DEBUG
```

Comment:
Wouldn't it be better for all this code to be part of `emitIssue1Instr`, and not here?

Reply:
Makes sense.